src/sentry/static/sentry/app/components/events/interfaces/threads/threadsSelector.tsx

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/prop-types */
```

Comment:
Do we need this if you accept the suggestion to type component differently?

Reply:
I believe not anymore :)